reporter/src/main/kotlin/reporters/AbstractNoticeReporter.kt

```diff
@@ -139,12 +141,31 @@ abstract class AbstractNoticeReporter : Reporter {
     private fun getLicenseFindings(
         ortResult: OrtResult,
         packageConfigurationProvider: PackageConfigurationProvider
-    ): Map<Identifier, LicenseFindingsMap> =
-        ortResult.collectLicenseFindings(packageConfigurationProvider, omitExcluded = true).mapValues { (_, findings) ->
-            findings.filter { it.value.isEmpty() }.keys.associate { licenseFindings ->
-                Pair(licenseFindings.license, licenseFindings.copyrights.map { it.statement }.toMutableSet())
-            }.toSortedMap()
+    ): Map<Identifier, LicenseFindingsMap> {
+        val concludedLicenses = ortResult.collectConcludedLicenses(omitExcluded = true)
+        val declaredLicenses = ortResult.collectDeclaredLicenses(omitExcluded = true)
+
+        val detectedLicenses = ortResult.collectLicenseFindings(packageConfigurationProvider, omitExcluded = true)
+            .mapValues { (_, findings) ->
+                findings.filter { it.value.isEmpty() }.keys.associate { licenseFindings ->
+                    Pair(licenseFindings.license, licenseFindings.copyrights.map { it.statement }.toMutableSet())
+                }.toSortedMap()
```

Comment:
Instead of `toSortedMap`and `toMutableMap` how about using `filterTo()`.

Reply:
True, but I only changed the indentation of these lines, and I don't want to change this as part of this PR.